src/main/kotlin/org/rust/toml/CargoTomlPsiPattern.kt

```diff
@@ -42,6 +42,18 @@ object CargoTomlPsiPattern {
         cargoTomlPsiElement<PsiElement>()
             .withParent(TomlKey::class.java)
 
+    fun inValueWithKey(key: String): PsiElementPattern.Capture<PsiElement> {
```

Comment:
Looks like we can simplify this pattern. For example, in the following way
```kotlin
fun inValueWithKey(key: String): PsiElementPattern.Capture<PsiElement> {
    return cargoTomlPsiElement<PsiElement>().inside(
        cargoTomlPsiElement<TomlKeyValue>()
            .withChild(
                cargoTomlPsiElement<TomlKey>().withText(key)
            )
    )
}
```

Reply:
Yeah, way better, thanks.
First time I do something like this and got lost in the API a bit.